ContinuousRegistration/Source/make_evaluation.py

```diff
@@ -10,6 +10,8 @@ def run(parameters):
     results = {}
     for team_name, blueprint_file_names in submissions.items():
         for blueprint_file_name in blueprint_file_names:
+            if not team_name in results:
```

Comment:
I could be wrong but it looks like `if not team_name in results` does not need to be checked for each `blueprint_file_name`, does it? If so, the `if ... continue` could move up, and could be placed directly after `for team_name, blueprint_file_names`.

Reply:
`results` is empty at the start, this if is just to initialize the dict at `results[team_name]` on the following line

Reply:
@kaspermarstal  I thought it could be faster to move the `if ...continue` two lines up, like this:

    for team_name, blueprint_file_names in submissions.items():
        if not team_name in results:
            continue
        for blueprint_file_name in blueprint_file_names:

But honestly I'm not really sure if that would give the same result. Just trying to understand the code!